lighthouse-core/lib/page-functions.js

```diff
@@ -218,6 +218,40 @@ function getNodeSelector(node) {
   return parts.join(' > ');
 }
 
+/**
+ * @param {Element} node
+ * @returns {string}
+ */
+/* istanbul ignore next */
+function getNodeTitle(node) {
+  if (node.tagName === 'HTML' || node.tagName === 'BODY') {
```

Comment:
it seems like there's something nice we could print for these...could we just always have a lowercase `tagName` as the fallback (here and if nothing is found below)? Or does that end up looking bad because it's right above `<body>` anyways

Reply:
Works for me, what do you think?

![Screenshot 2019-05-16 at 14 25 49](https://user-images.githubusercontent.com/1303660/57857215-90fa0a80-77e6-11e9-9b3e-0e36294ee325.png)